src/spikeinterface/qualitymetrics/pca_metrics.py

```diff
@@ -156,16 +160,16 @@ def compute_pc_metrics(
         )
         items.append(func_args)
 
-    if not run_in_parallel:
+    if not run_in_parallel and metric_names:
         units_loop = enumerate(unit_ids)
         if progress_bar:
-            units_loop = tqdm(units_loop, desc="calculate_pc_metrics", total=len(unit_ids))
+            units_loop = tqdm(units_loop, desc="calculate non nn pc_metrics", total=len(unit_ids))
```

Comment:
```suggestion
            units_loop = tqdm(units_loop, desc="calculate pc_metrics", total=len(unit_ids))
```
I would keep this simple ;)